package.json

```diff
@@ -4,10 +4,11 @@
   "description": "",
   "main": "index.js",
   "repository": {
+    "type": "git",
     "url": "https://github.com/oleg-koval/ya-skeleton"
   },
   "scripts": {
-    "start": "./bin/server | bunyan -o short",
+    "start": "nodemon ./bin/server | exec $LOGGER -o short",
```

Comment:
Is that for production as well?

https://github.com/remy/nodemon/blob/5e88b04eaa94ba72a18fca7d28517a0316fe4cb3/faq.md

> Note that I would not recommend using nodemon in a production environment - but that's because I wouldn't want it restart without my explicit instruction.

Check it out btw – https://github.com/wearereasonablepeople/maze-api/blob/2b14ae580dc347db228a79b067552f3182f90cd8/ecosystem.json#L12

For production, I would recommend to use `pm2`

Reply:
no, just for development for now

Reply:
there are some problems with pm2 two and f.e. heroku, we can discuss it offline